### DISCUSSION TITLE
Equilibrage égalités de fights

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1853] [Nouvelles idées]
Suggestion : Si la différence de pourcentage d'énergie est supérieure à X%, et qu'on arrive à la limite de tours, alors celui qui a mis le plus cher gagne. Ca évite les situations absurdes où un mec stall une game JUSTE pour gagner à quelques pvs alors que l'autre est full.

Ca garde le principe d'égalité, et ca met une pression en plus sur les tanks pour pas juste faire des fights inintéressent pour les non-tanks

**Comment by leuhvraiminoryo** [comment]
z'avez qu'à pouvoir nous tuer plus vite :slightly_smiling_face:
